docs/user/reference/openapi.yaml

```diff
@@ -88,14 +88,39 @@ components:
       additionalProperties: false
       description: Acknowledgement that a task has started, includes its ID
       properties:
-        taskName:
+        taskId:
           description: Unique identifier for the task
-          title: Taskname
+          title: Taskid
           type: string
       required:
-      - taskName
+      - taskId
       title: TaskResponse
       type: object
+    TrackableTask:
+      additionalProperties: false
+      description: A representation of a task that the worker recognizes
+      properties:
+        isComplete:
+          default: false
+          title: Iscomplete
```

Comment:
The autogenerated titles are a bit naff
Should be possible to make them a bit nicer on the Pydantic model
Field(title="The description of the item")